modules/@apostrophecms/i18n/index.js

```diff
@@ -9,6 +9,16 @@ const i18next = require('i18next');
 const i18nextHttpMiddleware = require('i18next-http-middleware');
 const fs = require('fs');
 
+const apostropheI18nDebugPlugin = {
+  type: 'postProcessor',
+  name: 'apostrophei18nDebugPlugin',
+  process(value, key, options, translator) {
+    // For ease of tracking down which phrases were
+    // actually passed through i18next
+    return `* ${value}`;
```

Comment:
This is marking strings that _have_ been localized? The idea being that one would look through the UI and find anything without the marker, look for the code it's coming from and add the `t` there? That sound helpful. I'm wondering we should separate i18next debugging and this since i18next debugging is its own thing. `APOS_SHOW_L10N`?

And I'm always for making things more obvious if they should be obvious. Maybe the `🌍` for this instead of an asterisk.

Reply:
Also needs a changelog update.